misc-utils/fincore.c

```diff
@@ -42,40 +43,93 @@
    e.g. 128MB on x86_64. ( = N_PAGES_IN_WINDOW * 4096 ). */
 #define N_PAGES_IN_WINDOW ((size_t)(32 * 1024))
 
+#ifndef HAVE_CACHESTAT
+
+#include <linux/types.h>
```

Comment:
We're trying to avoid kernel types and use standard userspace types if possible. Can't we use uint64_t here?